domains/auth/Auth.API/Tests/Services/OidcProviders/TestSignaturGruppen.cs

```diff
@@ -87,12 +91,67 @@ public void RedirectSuccess()
             ReturnUrl = "https://demo.energioprindelse.dk/dashboard"
         };
 
-        cryptography
-            .Setup(x => x.Encrypt(It.IsAny<string>()))
+        cryptographyFactory
+            .Setup(x => x.StateCryptography().Encrypt(It.IsAny<string>()))
             .Returns("foo=42");
 
         var res = signaturGruppen.CreateAuthorizationUri(state);
 
         Assert.Equal(expectedNextUrl, res.NextUrl);
     }
+
+    [Fact]
+    public async void FetchToken_Succes()
+    {
+        var expectedOidcTokenResponse = new OidcTokenResponse() { IdToken = "Test_id_token", AccessToken = "sd", ExpiresIn = 3600, TokenType = "Bearer", Scope = "openid nemid mitiduserinfo_token", UserinfoToken = "TEST_userinfo_token" };
```

Comment:
This is minor and i know there are very different opinions on this. But i think all these long lines makes it hard o quickly read a file. I would suggest using multiple lines